### ISSUE TITLE
Difference in styles between gists rendered on neo4j.com and portal.graphgist.org

### ISSUE THREAD

**Issue by johnymontana** [neo4j-examples/graphgist_portal#40]
The styles seem to be a bit inconsistent between graphgists rendered on neo4j.com and new gists rendered on the portal.

For example: http://neo4j.com/graphgist/8e1d6ff9-f7fa-4c99-9c5d-ecb94ba46d81

Do we want these to be consistent?

**Comment by cheerfulstoic** [comment]
I don't think we care.  The two sites haw two styles in general so it would make sense that the GraphGists would render differently.

On the technical note: neo4j.com uses the Foundation framework and the portal uses Semantic UI.  

I think that both renderings could still certainly use work.
